src/main/drivers/accgyro_fake.c

```diff
@@ -134,54 +108,45 @@ bool fakeGyroDetect(gyroDev_t *gyro)
 #ifdef USE_FAKE_ACC
 
 static int16_t fakeAccData[XYZ_AXIS_COUNT];
```

Comment:
I think we should leave it where it is for the moment. We can move it if and when we want more than one fake acc or gyro.

@cs8425 , can you test the simulator still works? If I haven't broken anything I think we should merge this in.

We can make further enhancements in a subsequent PR.